setup.py

```diff
@@ -5,133 +5,51 @@
         'Development Status :: 3 - Alpha',
-
-        # Indicate who your project is intended for
         'Intended Audience :: Developers',
         'Topic :: Software Development :: Build Tools',
-
-        # Pick your license as you wish (should match "license" above)
         'License :: OSI Approved :: BSD License',
         'Operating System :: OS Independent',
-        # Specify the Python versions you support here. In particular, ensure
-        # that you indicate whether you support Python 2, Python 3 or both.
         'Programming Language :: Python :: 2.7',
         'Topic :: Software Development :: Libraries :: Python Modules',
         'Topic :: Scientific/Engineering'
     ],
-
-    # What does your project relate to?
     keywords='Observations Data converter ODM2',
-
-    # You can just specify the packages manually here if your project is
-    # simple. Or you can use find_packages().
-
     packages=find_packages(exclude=['samplefiles', 'setup', 'tests*', 'Forms']),
-
-    # List run-time dependencies here.  These will be installed by pip when
-    # your project is installed. For an analysis of "install_requires" vs pip's
-    # requirements files see:
-    # https://packaging.python.org/en/latest/requirements.html
-
-    install_requires=install_reqs,
-    # install_requires=[
-    #     'pyodbc',
-    #     'six',
-    #     'sqlalchemy',
-    #     'shapely',
-    #     'pandas',
-    # ],
-
-    # List additional groups of dependencies here (e.g. development
-    # dependencies). You can install these using the following syntax,
-    # for example:
-    # $ pip install -e .[dev,test]
+    install_requires=install_requires,
     extras_require={
         'mysql': ['pymysql'],
         'postgis': ['psycopg2'],
-        'sqlite': ['pyspatialite >=3.0.0'], # need to look at: http://www.gaia-gis.it/spatialite-2.4.0-4/splite-python.html
-        'test': ['coverage'],
+        'sqlite': ['pyspatialite >=3.0.0'],
     },
-
-    # If there are data files included in your packages that need to be
-    # installed, specify them here.  If using Python 2.6 or less, then these
-    # have to be included in MANIFEST.in as well.
-    #
-    # package_data={
-    #     'sample': ['package_data.dat'],
-    # },
-
-    # Although 'package_data' is the preferred approach, in some case you may
-    # need to place data files outside of your packages. See:
-    # http://docs.python.org/3.4/distutils/setupscript.html#installing-additional-files # noqa
-    # In this case, 'data_file' will be installed into '<sys.prefix>/my_data'
-
-    # data_files=[('my_data', ['data/data_file'])],
-
-    # To provide executable scripts, use entry points in preference to the
-    # "scripts" keyword. Entry points provide cross-platform support and allow
-    # pip to create the appropriate form of executable for the target platform.
-
-    # entry_points={
-    #     'console_scripts': [
-    #         'sample=sample:main',
-    #     ],
-    # },
```

Comment:
Mostly cleanups. The comments are not really helpful and get in the way of reading the code here.